docs/CreateNewApp.md

```diff
@@ -1,22 +1,50 @@
 ## Overview
 
-In order to setup the RN-desktop project, execute these terminal commands:
+### Create project
+React native projects creating goes with the help of command line interface (`react-native-cli` package). `react-native-desktop` adds a modification to that package to support desktop platform.
 
+#### Use modified `react-native-cli` globally
+If all the projects you develop need desktop support, you can install modified command line interface globally by following [these steps](docs/InstallUpdatedReactNativeCLI.md)
+
+After that you create a new project:
+```sh
+react-native init DesktopSampleApp
+cd DesktopSampleApp
+```
+
+#### Use modified `react-native-cli` locally
+Alternative and more flexible way is to install original `react-native-cli`:
 ```sh
 npm install -g react-native-cli # Or use Yarn
+```
+And create project that uses a react-native-desktop repo:
+```sh
 react-native init DesktopSampleApp --version status-im/react-native-desktop
+```
+Since we used original `react-native-cli`, desktop-related files aren't in project yet. So we should initialize them:
+```sh
 cd DesktopSampleApp
 react-native desktop
```

Comment:
so this line is functionally equivalent to if we had initialized with the "extended" version of `react-native-cli` provided by the `react-native-desktop` repo?

Reply:
Yes, if we have original `react-native-cli` installed globally, then project files created only for ios and android, but `react-native` itself installed from our branch.
So when you execute `react-native desktop` from project folder, it invokes not globally installed `react-native-cli`, but the one provided by installed `react-native`, which can generate desktop files.